sycl/doc/syclcompat/README.md

```diff
@@ -1129,17 +1129,24 @@ spec, and so should be used with caution.
 namespace syclcompat {
 namespace experimental {
 
+#if defined(__AMDGPU__)
+// seq_cst currently not working for AMD
```

Comment:
Do we need a convention to start tracking `TODO:`/`FIXME:` in our headers and/or docs? Or can we handle this with Jira?

Reply:
Yes. As discussed, It's a good idea to have a `FIXME(@intel/syclcompat-lib-reviewers):` to track this kind of issues.